docs/installing.md

```diff
@@ -1,131 +1,93 @@
-# Installation
+# Percona Software Installation
 
+We recommend to install Percona software using the corresponding package manager
+for your system:
 
-It is recommended to install Percona software
-using the corresponding package manager for your system:
-
-
-* **apt** for [Debian and Ubuntu](#install-on-debian-and-ubuntu)
-* **yum** for [Red Hat Enterprise Linux and derivatives
-(including CentOS, Oracle Linux, Amazon Linux AMI, etc.)](#install-on-red-hat-enterprise-linux-and-centos)
+* **apt** for Debian and Ubuntu Linux
+* **yum** for Red Hat Enterprise Linux and compatible derivatives (including CentOS, Oracle Linux, Amazon Linux AMI, etc.)
 
 Find information about supported platforms on the [Percona Release Lifecycle Overview](https://www.percona.com/services/policies/percona-software-support-lifecycle#support).
 
-Alternatively, you can [download percona-release from Percona website](#download-from-percona-website)
-
-=== "Install on Debian and Ubuntu"
+=== "Installation on Debian and Ubuntu Linux"
 
-    If you are running a DEB-based distribution, such as Debian or Ubuntu,
-    use the **apt** package manager to install the `percona-release`
-    official package:
+    If you are running a DEB-based distribution, such as Debian or Ubuntu Linux, use
+    the **apt** package manager to install the `percona-release` official package:
 
-    !!! admonition "Prerequisites"
+    1. Install the `wget` download utility if it's not installed already:
 
-        In Linux distributions that rely on `dpkg`, the packages `wget`, `gnupg2`, `curl` and `lsb-release` are already installed. However, these packages may be missing from Docker base images. In this case, install them manually before running `dpkg`:
-
-        ```
+        ``` sh
         sudo apt update
-        sudo apt install -y wget gnupg2 curl lsb-release 
+        sudo apt install wget 
```

Comment:
this set of packages is still needed on debian based systems

Reply:
`lsb-release` is a dependency of `percona-release` and will thus be installed automatically when using `apt` for installing it. `curl` does not seem to be used by `percona-release` and I did not notice any error in my tests about it not being installed.

Reply:
I'll keep the packages on the command line for now until `percona-release` has been updated to include these dependencies by default.